### PR TITLE
Use new call centre accordion in save journey

### PR DESCRIPTION
**Pull request by andrewHEguardian**
## What does this change?

Use the new call centre accordion in the saves journey.

## How to test

See storybook, or cancel a membership.

## Images

image
image

## Accessibility

-   [ ] [Tested with screen reader](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#screen-reader)
-   [ ] [Navigable with keyboard](https://github.com/guardian/accessibility/blob/main/people-and-technology/02-physical.md#keyboard)
-   [ ] [Colour contrast passed](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#contrast)
-   [ ] [The change doesn't use only colour to convey meaning](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#use-of-colour)

### CODE REVIEW

File: client/components/shared/CallCentreAccordion.stories.tsx [hunk 1]
```diff
@@ -1,9 +1,32 @@
...
 	component: CallCentreAccordion,
 } as Meta<typeof CallCentreAccordion>;
 
-export const Default: StoryObj = {};
+export const Closed: StoryObj = {};
+
+export const Open: StoryObj = {
+	args: { showEmailAddress: true },
+	play: async ({ canvasElement }: { canvasElement: HTMLElement }) => {
+		// There is a useEffect within AccordionRow that sets 'isBrowser' that isn't being run in storybook
```
Comment (andrewHEguardian) [inline client/components/shared/CallCentreAccordion.stories.tsx:17]: I had to do some investigation to figure out why this didn't work, and found the `sleep(1)` solution on this [issue](https://github.com/storybookjs/storybook/issues/18258). A bit of a hacky workaround so maybe won't use lots of `play` functions 😞

Comment (charleycampbell) [reply client/components/shared/CallCentreAccordion.stories.tsx:17]: It is a wee bit but I like it! We could add an additional code comment explaining the sleep(1) is required due a bug....

**Review by charleycampbell — DISMISSED** [review]
Just one suggestion around adding a comment to the "hacky" part but other than that nice work!

**Review by charleycampbell — APPROVED** [review]
Perfect code comment 🖋️

**Pull request merged**